### ISSUE TITLE
Screen reader announces 'blank' when down key is pressed on dropdown with scan mode

### ISSUE THREAD

**Issue by amitavak** [microsoft/fluentui#21166]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/amitavak/pen/mdBKeNK

#### Actual behavior:

Screen reader announces 'blank' when down key is pressed on dropdown with scan mode

#### Expected behavior:

Screen reader should not announces 'blank' when down key is pressed on dropdown with scan mode

**Comment by AndrewCraswell** [comment]
Looks like this is a duplicate of a bug I filed last week, same issue. Our repro sandbox is identical, which shows there's an issue when the dropdown is right above a checkbox.
https://github.com/microsoft/fluentui/issues/21025

**Comment by gouttierre** [comment]
@amitavak - Thanks for filing this issue with us. To set expectations, the developers will review this issue once capacity allows.

@spmonahan - Would you be able to confirm if this is a regression, or if this behavior is an issue? 
cc @smhigley

**Comment by tringakrasniqi** [comment]
@amitavak  and @AndrewCraswell - thank you for filing this, however this seems to be normal behaviour of Narrator when it encounters isolated white space of any kind and is not an issue with the components. I confirmed this with @smhigley too, so I will go ahead and close these two issues, thanks again.

**Comment by AndrewCraswell** [comment]
@tringakrasniqi can you clarify which isolated white space you are referring to causing the issue? Given the Codesandbox, I didn't see any isolated whitespace which wasn't directly part of the component itself.
